pkg/sql/information_schema.go

```diff
@@ -498,16 +513,15 @@ https://www.postgresql.org/docs/9.5/infoschema-columns.html`,
 					tree.DNull, // maximum_cardinality
 					tree.DNull, // dtd_identifier
 					tree.DNull, // is_self_referencing
-					//TODO: Need to update when supporting identiy columns (Issue #48532)
-					noString,                          // is_identity
-					tree.DNull,                        // identity_generation
-					tree.DNull,                        // identity_start
-					tree.DNull,                        // identity_increment
-					tree.DNull,                        // identity_maximum
-					tree.DNull,                        // identity_minimum
-					tree.DNull,                        // identity_cycle
-					yesOrNoDatum(column.IsComputed()), // is_generated
-					colComputed,                       // generation_expression
+					yesOrNoDatum(column.IsGeneratedAsIdentity()), // is_identity
+					colGeneratedAsIdentity,                       // identity_generation
+					tree.DNull,                                   // identity_start
```

Comment:
can we fill out these other columns? the start, increment, max, min, and cycle should all be available on the sequence options

Reply:
hm, i see that it looks like we'd need to re-parse the `column.GetGeneratedAsIdentitySequenceOption()` string. let's not worry about it for right now